tox.ini

```diff
@@ -1,21 +1,27 @@
 [tox]
-envlist = py27, py35, py36
+envlist = py27, py38
 skip_missing_interpreters = true
-requires =
-    pip >= 20.1.1
-    virtualenv >= 20.0.21
 
 [testenv]
 whitelist_externals =
-    /bin/bash
-    /usr/bin/wget
-    /usr/local/bin/wget
-    /usr/bin/java
+    bash
+    java
+    rm
+    wget
+
 deps =
     -rrequirements.txt
     -rdev-requirements.txt
-commands =
-    wget -N {env:SWAGGER_URL:https://oss.sonatype.org/content/repositories/releases/io/swagger/swagger-codegen-cli/2.3.1/swagger-codegen-cli-2.3.1.jar}
-    java -jar swagger-codegen-cli-2.3.1.jar generate -i openapis/swagger.yaml -l python -o swagger_client
+
+install_command = python -m pip install --no-deps {opts} {packages}
+
+commands_pre =
+    wget --quiet -N {env:SWAGGER_URL:https://repo1.maven.org/maven2/io/swagger/codegen/v3/swagger-codegen-cli/3.0.25/swagger-codegen-cli-3.0.25.jar} -O swagger-codegen-cli.jar
```

Comment:
Could you also update the swagger-codegen-cli URL in `docker-compose-ci.yaml`? The Jenkins build is failing with the older version. I've updated the repo settings so that the Jenkins checks appear.